OneCommunityGlobal/HighestGoodNetworkApp
# Olga design projects summary view

# Description
Design of BM dashboard displaying projects view page and its basic functionality. Here is the Figma prototype:

![Screen Shot 2023-10-06 at 2 52 53 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/107726913/ec5e414e-094e-4294-bae1-17cb0681c5a8)


## Main changes explained:
Created new components in BMDashboard/Projects, updated BMDashboard.jsx design and populated it with dummy data.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. go to localhost:3000 and login with any user
4. go to localhost:3000/bmdashboard and log in again with the same user
5. verify that you are taken to the building and inventory management view below:

![Screen Shot 2023-10-06 at 2 57 38 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/107726913/0b4d8d83-ae5b-4c08-ac1f-4bf03abe5f3c)

6. verify that:
 - projects in the select dropdown are the same as the projects displayed in the blue rectangles. 
 - when selecting a project from the dropdown menu and clicking on the button you are redirected to  a new page (localhost:3000/bmdashboard/projects/:projectId) (that page is blank for now except for a project id and a welcome message)
 - page is responsive to different screens
